doc/radosgw/adminops.rst

```diff
@@ -1086,17 +1089,16 @@ Create Key
 ==========
 
 Create a new key. If a ``subuser`` is specified then by default created keys
-will be swift type. If only one of ``access-key`` or ``secret-key`` is provided the
-committed key will be automatically generated, that is if only ``secret-key`` is
-specified then ``access-key`` will be automatically generated. By default, a
+will be swift type. If only one of ``access-key`` or ``secret-key`` is provided an
```

Comment:
Though this is out of scope for this PR I noticed that both of the keys needed to be included in a create new key request or else the operation would not go through. I get a `InvalidSecretKey` and `InvalidAccessKeyID` when either are empty in a request.